### PR TITLE
fix: Return page width and height from document

### PR DESCRIPTION
**Pull request by vitormattos**
# Type of pull request

* [ ] Bug fix (involves code and configuration changes)
* [ ] New feature (involves code and configuration changes)
* [x] Documentation update
* [ ] Something else

# About

Some documents could have pages without MediaBox, at this case we need to get the MediaBox from document and not from a specific page.

### CODE REVIEW

**Comment by k00ni** [comment]
Thank you, good addition to the documentation @vitormattos

File: doc/Usage.md [hunk 1]
```diff
@@ -224,6 +224,11 @@ $pages = $pdf->getPages();
 $mediaBox = [];
 foreach ($pages as $page) {
     $details = $page->getDetails();
+    // The page could haven't the Mediabox, then we get this information from document
```
Comment (k00ni) [inline doc/Usage.md:227]: The comment doesn't reflect the code in my opinion. Here is an alternative:

```suggestion
    // If Mediabox is not set in details of current $page instance, get details from the header instead
```

Comment (vitormattos) [reply doc/Usage.md]: Thanks! English is not my main language.

**Review by k00ni — APPROVED** [review]
:+1:

**Pull request merged**